core/src/main/java/org/radargun/config/FlightRecorder.java

```diff
@@ -17,6 +17,9 @@
 
    @Property(doc = "Flight Recorder Options")
    private String flightRecorderOptions;
+   
+   @Property(doc = "Is this an Oracle JVM? Default is false.")
+   private boolean isOracle = false;
```

Comment:
This is something you can autodetect, see `java.vendor` (regrettably Oracle Corporation for both commercial and OpenJDK) and `java.runtime.name` (that differs).

Reply:
Thanks, @rvansa! Missed the `java.runtime.name` property

Reply:
Updated!